domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/build.gradle

```diff
@@ -8,6 +8,8 @@ ext.jacoco_additional = [
     prefixedProject('core-robstoll-lib-jvm'),
 ]
 
+spek.version = "$spek2_version"
```

Comment:
is this really required? I might be wrong but I thought spek2 is already available

Reply:
Spek2 is not available and compilation fails without this line.

Reply:
Done.

Reply:
Done.